tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -865,19 +865,19 @@ def visit_call(self, node):
 class ClientListMethodsUseCorePaging(BaseChecker):
     __implements__ = IAstroidChecker
 
-    name = "client-list-methods-use-paging"
+    name = "client-paging-methods-use-list"
     priority = -1
     msgs = {
         "C4733": (
-            "Operations that return collections should return a value that implements the Paging protocol. See details:"
+            "Operations that return collections should return a value that implements the Paging protocol and be prefixed with list_ or _list_. See details:"
```

Comment:
What's the case for `_list_`?

Reply:
Case for _list was after talking to Johan, even if it is a private _list function we still want it to conform to the SDK rules of returning an iterable, or if a private function returns an iterable we want it to be _list prefixed